community/STEERING-COMMITTEE.md

```diff
@@ -1,61 +1,118 @@
 
-In case of extended absence, an absent member will be considered to be in
-agreement after 8 days have passed since a proposal was accepted by all present
-members. An absent member may appoint a single delegate during absences.
+Members of the committee may step down at any time. When a member steps down, the
+company who holds the seat shall nominate a new individual for the seat.
 
-<!-- TODO ## Committee Meeting -->
+Changes to the number of seats, which company seats are allocated to, and nominations
+to the committee are confirmed by majority vote of the committee members.
 
-## Committee Members
+## Frequently asked questions
 
-The members of the Steering Committee are shown below. Membership in the SC is
-determined by current level of contribution to the project. Contribution is
-periodically reviewed to ensure proper recognition.
+_Why are seats company held?_
 
-| &nbsp;                                                   | Member         | Company | Profile                                  |
-| -------------------------------------------------------- | -------------- | ------- | ---------------------------------------- |
-| <img width="30px" src="https://github.com/dewitt.png">   | DeWitt Clinton | Google  | [@dewitt](https://github.com/dewitt)     |
-| <img width="30px" src="https://github.com/mchmarny.png"> | Mark Chmarny   | Google  | [@mchmarny](https://github.com/mchmarny) |
-| <img width="30px" src="https://github.com/isdal.png">    | Tomas Isdal    | Google  | [@isdal](https://github.com/isdal)       |
+The committee was created as the project was in its infancy, in order to tackle governance
+and overall project strategy. Because of the nature of the project and funding required,
+it was decided that strong corporate leadership was necessary for the project to ensure
+velocity.
+
+_What happens if someone leaves the company they represent?_
+
+In that case, the company will nominate a new representative for the seat. The nominee
+shall be confirmed by the standard process as called out above.
+
+_What happens if someone leaves the project or decides to leave the committee?_
+
+As above, the same would apply. The company who holds the seat will nominate a new
+representative to hold the seat. The nominee shall be confirmed by the standard process,
+called out above.
+
+_Is this steering model permanent?_
+
+The governance of Knative is an open, living document, and will continue to evolve as
+the community and project change. We expect over time we will adapt the way we run this
+committee, based on feedback from the community.
+
+_How are steering committee decisions reached?_
+
+The steering committee desires to always reach consensus. Decisions are made in
+meetings when a quorum of the members are present and may pass with majority
+vote. Quorum is considered reached when at least half of the members are present.
+
+The committee's charter may be changed by majority consent of committee members.
```

Comment:
This is problematic insofar as committee members are reasonably likely to vote en bloc some of the time. Under the initial allocation, Google would be able to amend the KSC rules unilaterally.

I'd suggest a two-part rule: A majority of members, with votes in favour cast by committee members from at least two separate companies. As the committee grows and becomes more plural, this could be be updated to a stricter rule: a majority of votes and a majority of companies.

Reply:
the problem is not in the quorum and majority vote, the problem is in allocating seats to vendors and not individuals.

You could say: Google, IBM, RH and Pivotal hold "seeding" seats but the committee is elected from community nominations. See maximal representation in Kubernetes steering election process: https://github.com/kubernetes/steering/blob/master/elections.md#maximal-representation

Reply:
I don't object to this idea, though I assume it means there would need to be a some kind of secretariat to manage the elections process. Kubernetes gets this through the CNCF.

Reply:
I hear this feedback loud and clear, but we're still in the early days of this project and rely on organizations to provide the majority of support to the project.  I also don't see us having the machinery nor human-power that would be necessary to run the full process that k8s uses for managing their project and steering committee elections.

I've based the proposed process here off what's working for Istio today, which is in a similar state to where we are with Knative. As our project matures, I think adopting more of the k8s processes may be worthwhile.

Given the importance of the organization backing for this project during our growth phase, I see it as necessary and appropriate to have the seats held by organizations instead of individuals, but I understand the pushback.